apps/module-scan/schema.sql

```diff
@@ -1,5 +1,6 @@
 create table batches (
   id varchar(36) primary key,
+  label text,
```

Comment:
I think we should have a `sequence integer autoincrement` in addition to this value. If `label` is set, then it's a custom value. Otherwise use `Batch ${sequence}`. Or, if we don't want to overcomplicate it just replace `label`.

Reply:
Yeah fair point.